cozy/target_syntax.py

```diff
@@ -95,3 +95,8 @@ def EDisjoint(xs, ys):
 SMapPut    = declare_case(Stm, "SMapPut", ["map", "key", "value"])
 SMapDel    = declare_case(Stm, "SMapDel", ["map", "key"])
 SMapUpdate = declare_case(Stm, "SMapUpdate", ["map", "key", "val_var", "change"]) # val_var is EVar
+
+# orderd multiset
```

Comment:
These should probably be defined in `ordereds.py` with the rest of the related functionality.